Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -387,7 +387,7 @@ This block implements a rule-based tuning method for a PI or a PID controller, w
 steps:
```

Comment:
`This block implements a PI or PID controller with the control gains being tuned by a rule-based method. The tuning method has following steps:`